src/StateSmith/Output/Sim/HtmlRenderer.cs

```diff
@@ -343,20 +348,23 @@ public static void Render(StringBuilder stringBuilder, string smName, string moc
         function highlightEdge(edgeId) {
             var edge = document.getElementById(edgeId);
             if (edge) {
-                edge.style.stroke = 'red';
+                edge.style.stroke = 'yellow';
```

Comment:
Agree the yellow is a bit attention-getting. I'm definitely open to fine-tuning to something less bright if you have suggestions. I'd love to have a single color-family that represents "active", whether for transitions or states, although we might tweak that color slightly between the two (eg. dial down the state color a bit since it covers more surface area than the transition color).

Agree a css style would be better. Will look at whether !important overrides the css matcher that's already there.

Reply:
oh, it might also be worse for you because you use darkmode. Might make sense to have a different color for darkmode